lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -372,6 +372,18 @@ module Resources
         'op': 'replace',
         'path': '/status',
         'value': 'cancelled'
+      },
+      {
+        'op': 'add',
+        'path': '/cancelationReason',
+        'value': {
+          'coding': [
+            {
+              'system': 'http://terminology.hl7.org/CodeSystem/appointment-cancellation-reason',
+              'code': 'oth-err'
+            }
+          ]
+        }
```

Comment:
since this is a new operation, this should be created separately from the status patch.

Reply:
Added new operation example on this commit: https://github.com/cerner/fhir.cerner.com/pull/593/commits/4b69d086d4851d13bf49acff5e5f4b7c8106dba2